packages/xo-web/src/common/select-objects.js

```diff
@@ -358,10 +360,26 @@ export const SelectPool = makeStoreSelect(
 
 export const SelectSr = makeStoreSelect(
   () => {
-    const getSrsByContainer = createGetObjectsOfType('SR')
-      .filter((_, { predicate }) => predicate || isSrWritable)
-      .sort()
-      .groupBy('$container')
+    const getSrsByContainer = createSelector(
+      createGetObjectsOfType('SR')
+        .filter((_, { predicate }) => predicate || isSrWritable)
+        .sort(),
+      srs => {
+        for (const sr1 of srs) {
+          for (const sr2 of srs) {
+            if (sr1.name_label === sr2.name_label && sr1.id !== sr2.id) {
+              sr1.name_label = `(${getObjectById(sr1.$container).name_label}) ${
```

Comment:
You may want to use a selector instead of getting the container with an explicit `store.getState()`.
```js
const getContainerFinder = createSelector(
  createGetObjectsOfType('host'),
  createGetObjectsOfType('pool'),
  (hosts, pools) => id => hosts[id] || pools[id]
)
```